Realm/Realm/Handles/SharedRealmHandle.cs

```diff
@@ -438,7 +449,7 @@ private static unsafe void HandleOpenRealmCallback(IntPtr taskCompletionSource,
             }
         }
 
-        [MonoPInvokeCallbackAttribute(typeof(NativeMethods.OnBindingContextDestructedCallback))]
+        [MonoPInvokeCallback(typeof(NativeMethods.OnBindingContextDestructedCallback))]
```

Comment:
I haven't had much luck in finding the difference between these 2. It seems that MonoPInvokeCallback comes up in Unity conversations, while with _Attribute generally leads to Microsoft docs. Could you elaborate on the differences?

Reply:
There's no difference between these. The compiler allows you to omit the `Attribute` suffix whenever you apply an attribute to reduce the verbosity. Ref https://stackoverflow.com/a/6710431/1649102.